holmes/plugins/toolsets/internet.py

```diff
@@ -1,79 +1,101 @@
 import re
+import os
 import logging
 
-from typing import Any
-from holmes.core.tools import Tool, ToolParameter, Toolset, ToolsetCommandPrerequisite
+from typing import Any, Optional, Tuple
+
+from requests import RequestException, Timeout
+from holmes.core.tools import Tool, ToolParameter, Toolset, ToolsetTag
 from markdownify import markdownify
-from playwright.sync_api import Error as PlaywrightError
-from playwright.sync_api import TimeoutError as PlaywrightTimeoutError
-from playwright.sync_api import sync_playwright
 from bs4 import BeautifulSoup
 
-# TODO: change and make it holmes
-USER_AGENT_STR = (
-    "Mozilla/5.0 (X11; Linux x86_64; rv:128.0; holmesgpt;) Gecko/20100101 Firefox/128.0"
-)
-PAGE_LOAD_TIMEOUT_SECONDS = 60000
-
-
-def scrape_with_playwright(url):
+import requests
 
-    with sync_playwright() as p:
-        try:
-            browser = p.firefox.launch()
-        except Exception as e:
-            logging.error(str(e))
-            return None, None
+# TODO: change and make it holmes
```

Comment:
What is this TODO? i see it contains holmesgpt so not sure i fully understand what it means :)

Reply:
This is a reminder to potentially change the user agent. Bots or automated systems should typically not use a browser user agent but something specific to them. I used a user agent b/c some websites don't return the html unless the user agent is a browser.